backend/src/akvo/lumen/auth.clj

```diff
@@ -37,22 +37,40 @@
   (-> (response/response "Not authorized")
       (response/status 403)))
 
+(def service-unavailable
+  (-> (response/response "Service Unavailable")
+      (response/status 503)))
+
+(def internal-server-error
+  (-> (response/response "Internal server errror")
+      (response/status 500)))
+
+(defn api-authz?
+  "Feature flag predicate function. Add trailing $auth to given_name to enable
+  API authz"
+  [{:strs [family_name] :as jwt-claims}]
+  (if (nil? family_name)
```

Comment:
maybe use `some->` to avoid the if condition?

Reply:
`and` returns the last true value
`(and (nil? family_name) (string/ends-with? family_name "$auth"))`

example `(and 1 2)` => 2